robo.yml

```diff
@@ -78,3 +78,8 @@ data:
     graph: "http://publications.europa.eu/resource/dataset/sdg"
     url: "http://publications.europa.eu/resource/cellar/5ea37239-0495-11eb-a511-01aa75ed71a1.0001.01/DOC_1"
     format: "rdf"
+
+  - name: "human-sex"
+    graph: "http://publications.europa.eu/resource/dataset/human-sex"
+    url: "http://publications.europa.eu/resource/cellar/12e3b382-278b-11e8-ac73-01aa75ed71a1.0001.03/DOC_1"
```

Comment:
I see humansexes-skos.rdf in page, but didn't see anything like that uri there, which is the cellar uri and how do I decode it? is there any documentation I can read ? thx

Reply:
You right click it and copy the link, then run a url decode (there are websites online) so that you decode the cellarURI parameter and copy that. There's no documentation.